src/embed/main.js

```diff
@@ -79,19 +87,39 @@ function onVideoTap() {
   worldRenderer.videoProxy.play();
   hidePlayButton();
 
+  // TODO(hoch): refactor this.
+  omnitoneDecoder.initialize().then(function () {
+    console.log('initialized');
+    videoElement.play();
+  }, function (error) {
+    console.log(error);
+  });
+
   // Prevent multiple play() calls on the video element.
   document.body.removeEventListener('touchend', onVideoTap);
 }
 
 function onRenderLoad(event) {
   if (event.videoElement) {
+
+    // TODO: activate this only when it's in the 'spatial' audio mode.
```

Comment:
Either there is a way to infer that we're dealing with a spatial audio track (maybe make assumption based on number of audio tracks?) or we need to implement a new GET param to trigger spatial mode.